ui/redux/actions/comments.js

```diff
@@ -889,6 +889,7 @@ function doCommentModToggleBlock(
       );
     }
 
+    const isBlocked = selectChannelIsBlocked(state, commenterUri);
```

Comment:
I see that all the 3 unblock functions have been consolidated to this one, but they are separated because their block list is not the same.  I think this change will break Mod and Admin toggling, most likely making the state out of sync with the GUI.

I couldn't confirm for sure though as Beamer broke `moderation` today -- will look when it is fixed.